src/ganj/HumanRiddleMaker.java

```diff
@@ -0,0 +1,30 @@
+package ganj;
+
+import java.util.Scanner;
+
+public class HumanRiddleMaker implements RiddleMaker {
```

Comment:
You missed the point of this class. Human riddle maker should use human mind to store the secret number. Nobody should know secret number except the riddle maker itself. And then when some other player tries to guess it the system should ask the human to compare suggested number and the secret one. Then the player should enter `<`, `>` or `=` depending on the comparison result.